Description: Default to PIC on arm
 Otherwise the armhf build fails now that PIE is enabled by default.
Author: Mehdi Dogguy <mehdi@debian.org>
Date: Sun, 6 Nov 2016 10:36:00 +0100
Bug-Debian: https://bugs.debian.org/837359
Forwarded: TODO

---
 asmcomp/arm/arch.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/asmcomp/arm/arch.ml b/asmcomp/arm/arch.ml
index d93c1e0..3541106 100644
--- a/asmcomp/arm/arch.ml
+++ b/asmcomp/arm/arch.ml
@@ -56,7 +56,7 @@ let (arch, fpu, thumb) =
     end in
   (ref def_arch, ref def_fpu, ref def_thumb)
 
-let pic_code = ref false
+let pic_code = ref true
 
 let farch spec =
   arch := (match spec with
